changelogs/fragments/280-vlan_expander.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
```suggestion
minor_changes:
```
Major changes are backwards incompatible changes to existing features, which this is not

Reply:
Thank you.  I committed it.